build/musicbrainz-dev/scripts/fetch-dump.sh

```diff
@@ -98,6 +98,59 @@ then
 	exit 64 # EX_USAGE
 fi
 
+# Show information about signing up for data use
+
+if [[ $TARGET != sample &&
+	! -a "$DB_DUMP_DIR/.for-commercial-use" &&
+	! -a  "$DB_DUMP_DIR/.for-non-commercial-use"
+]]
+then
+	prompt=$(cat <<-EOQ
+		The data you are about to download is provided by the MetaBrainz Foundation.
+		Are you planning to use this data for commercial or business purposes?
+		(y/n)
```

Comment:
Should we ask this every X time in case they originally signed up as non-commercial but are now doing commercial stuff? (so, if they still say no, skip the rest, if they say yes, ask them to sign up as commercial?) Or not worth it?

Reply:
Probably fair enough :)